### ISSUE TITLE
Red team base defences and logisitics centre not spawning after capture [MIZ 3.6.1]

### ISSUE THREAD

**Issue by madrabbit711** [ModernColdWar/RedStormRising#152]
Russians (red team) captured Beslan, Senaki and Kutasi after restart, but neither base defences, nor logisitics centre spawned.

Both base defences and logisitics centre spawn is working for blue team.

RSR_Dynamic_PvP_Spring_AM_Clear_V3.6.1

**Comment by Winston2108** [comment]
Not side specific, but in either MOOSE or more likely how we're using it:
```
2020-02-12 15:08:13.553 INFO    SCRIPTING: Utils|activateBaseDefences|35: Activating base defences for red base Senaki-Kolkhi within 5000m
2020-02-12 15:08:13.554 INFO    SCRIPTING: Error in SCHEDULER function:[string "C:\Users\Administrator\Saved Games\DCS.openbeta_server\Scripts\RSR\Moose.lua"]:35568: attempt to index a nil value
2020-02-12 15:08:13.554 INFO    SCRIPTING: stack traceback:
    [string "C:\Users\Administrator\Saved Games\DCS.openbeta_server\Scripts\RSR\Moose.lua"]:6740: in function <[string "C:\Users\Administrator\Saved Games\DCS.openbeta_server\Scripts\RSR\Moose.lua"]:6736>
    [string "C:\Users\Administrator\Saved Games\DCS.openbeta_server\Scripts\RSR\Moose.lua"]:35568: in function 'GetVec3'
    [string "C:\Users\Administrator\Saved Games\DCS.openbeta_server\Scripts\RSR\bases.lua"]:39: in function 'IteratorFunction'
```

**Comment by Winston2108** [comment]
Pushed a fix for this last night which moved the problem very slightly.  Just pushed another fix now which may work, and if not, logs the name of the "bad" group so I could do some diagnosis.

**Comment by madrabbit711** [comment]
Old issue that seems fixed in RSR 4.0
